demo/test.cc

```diff
@@ -39,3 +45,34 @@ TEST(Demo, OK) {
   EXPECT_EQ(200, static_cast<int>(response.code));
   EXPECT_EQ("OK\n", response.body);
 }
+
+TEST(Demo, NoPointsYet) {
+  Singleton<DemoServer>();
+  const auto response = HTTP(GET("localhost:2015/demo_id"));
+  EXPECT_EQ(200, static_cast<int>(response.code));
+  EXPECT_EQ("{\"state\":{\"points\":[]}}\n", response.body);
+}
+
+TEST(Demo, AddThreePoints) {
+  Singleton<DemoServer>();
+  // TODO(dkorolev): Support JSON parsing on the server side, like this:
+  // HTTP(POST("localhost:2015/demo_id", JSON(State::Point(+0.25, -0.25, true)), "application/json")).body);
+  // TODO(dkorolev): Should be able to HTTP(POST(...)) JSON-s directly, w/o cerealize::JSON(), right?
+  // TODO(dkorolev): How about POST-s without HTML body? Do we need those extra parameters?
```

Comment:
Thanks for the note!

Here's my view: One thing I don't want to do in Bricks (yet, and, possibly, ever) is to parse querystring-encoded data from BODY.

I might be wrong, but it feels that querystring in the URL and JSON in body are just enough.

Here's how it looks now. Please take another look. And happy to discuss!